docs-src/content/using-cubic-sdk/recognize.md

```diff
@@ -71,16 +73,19 @@ func main() {
 
 }
 ```
+
 {{% /tab %}}
 
 {{% tab "Python" %}}
+
 ``` python
 
 import cubic
 
 serverAddress = '127.0.0.1:2727'
+insecure = True # set to true if server does not use TLS
 
-client = cubic.Client(serverAddress)
+client = cubic.Client(serverAddress, insecure=insecure)
```

Comment:
As far as aesthetics go, this might look a little nicer if it was `insecure=True`, then skip setting the variable above. Or set the variable but use `cubic.Client(serverAddress, insecure)`